include/filters/filter_chain.h

```diff
@@ -198,20 +198,10 @@ class FilterChain
         }
 
 
-	//CHeck for backwards compatible declarations
 	if (std::string(config[i]["type"]).find("/") == std::string::npos)
 	  {
-	    ROS_WARN("Deprecation Warning: No '/' detected in FilterType, Please update to 1.2 plugin syntax. ");
-	    std::vector<std::string> libs = loader_.getDeclaredClasses();
-	    for (std::vector<std::string>::iterator it = libs.begin(); it != libs.end(); ++it)
-	      {
-		size_t position =  it->find(std::string(config[i]["type"]));
-		if (position != std::string::npos)
-		  {
-		    ROS_WARN("Replaced %s with %s", std::string(config[i]["type"]).c_str(), it->c_str());
-		    config[i]["type"] = *it;
-		  }
-	      }
+	    ROS_ERROR("Bad filter type %s. Filter type must be of form <package_name>/<filter_name>", config[i]["type"]);
```

Comment:
This leads to a type warning with `ROS_ERROR`, `std::string(config[i]["type"])` should be used (as before).

```
/home/mathias/build/lunar/src/ros_canopen/canopen_motor_node/src/handle_layer.cpp:174:81:   required from here
/opt/ros/lunar/include/ros/console.h:346:26: warning: format ‘%s’ expects argument of type ‘char*’, but argument 8 has type ‘XmlRpc::XmlRpcValue*’ [-Wformat=]
     ::ros::console::print(filter, __rosconsole_define_location__loc.logger_, __rosconsole_define_location__loc.level_, __FILE__, __LINE__, __ROSCONSOLE_FUNCTION__, __VA_ARGS__)
     ~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/opt/ros/lunar/include/ros/console.h:349:5: note: in expansion of macro ‘ROSCONSOLE_PRINT_AT_LOCATION_WITH_FILTER’
     ROSCONSOLE_PRINT_AT_LOCATION_WITH_FILTER(0, __VA_ARGS__)
     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/opt/ros/lunar/include/ros/console.h:379:7: note: in expansion of macro ‘ROSCONSOLE_PRINT_AT_LOCATION’
       ROSCONSOLE_PRINT_AT_LOCATION(__VA_ARGS__); \
       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
/opt/ros/lunar/include/ros/console.h:561:35: note: in expansion of macro ‘ROS_LOG_COND’
 #define ROS_LOG(level, name, ...) ROS_LOG_COND(true, level, name, __VA_ARGS__)
                                   ^~~~~~~~~~~~
/opt/ros/lunar/include/rosconsole/macros_generated.h:214:24: note: in expansion of macro ‘ROS_LOG’
 #define ROS_ERROR(...) ROS_LOG(::ros::console::levels::Error, ROSCONSOLE_DEFAULT_NAME, __VA_ARGS__)
                        ^~~~~~~
/opt/ros/lunar/include/filters/filter_chain.h:203:6: note: in expansion of macro ‘ROS_ERROR’
      ROS_ERROR("Bad filter type %s. Filter type must be of form <package_name>/<filter_name>", config[i]["type"]);
```